src/pages/css/css3-transitions/index.md

```diff
@@ -3,13 +3,124 @@ title: CSS3 Transitions
 ---
 ## CSS3 Transitions
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/css3-transitions/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Using CSS3 **Transitions** can be useful if you want your app or your web page to be more dynamic and good looking.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+Indeed, transitions allow you to change property (`width`, `color`, ...) values in a **smooth** way.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+The `transition` property is a shorthand property for `transition-property`, `transition-duration`, `transition-timing-function`, `transition-delay`, the syntax is the following :
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+```css
+transition: transition-property transition-duration transition-timing-function transition-delay
+```
+
+For example : 
+
+```
+transition: width 2s ease-in-out 1s;
+```
+
+### Description of the properties
+
+#### `transition-property`
+
+Specify the **name** of property to which you should apply a transition : 
+
+* `background-color`
+* `color`
+* `width`
+* `height`
+* `margin`
+* `border-radius`
+* And so on !
+
+For example : 
+
+```
+transition-property: width; /* means the transition applies on the width */
+```
+
+#### `transition-duration`
+
+Specify the **number of seconds or milliseconds** the transition should **take** : 
+
+For example :
+
+```
+transition-duration: 2s /* means the transition effect last 2s */
+```
+
+#### `transition-timing-function`
+
+Specify the **speed curve** of the transition effect. Thus, you can change your **transition's speed over its duration**. 
+
+Here are the most used values :
+
+* `linear`
+* `ease`
+* `ease-in`
+* `ease-out`
+* `ease-in-out`
+* `cubic-bezier(n, n, n, n)`
+
+For example :
+
+```css
+transition-timing-function: linear
+```
 
+N.B : All the values above are in fact specifics `cubic-bezier`. `linear`, for instance, is equivalent to `cubic-bezier(0.25,0.1,0.25,1)`
```

Comment:
N.B??

Reply:
N.B means : "Nota Bene". It means here "Take careful notice of the following"